backend/Dockerfile.worker

```diff
@@ -1,10 +1,14 @@
-FROM node:18-bullseye as build
+FROM node:18-alpine3.17 as build
```

Comment:
Should we switch back to alpine?

Reply:
We could as next step after this check-in.

Reply:
Oops. I meant to ask -- "should we switch back to bullseye"?

Given that Matt B did a bunch of work on that in the past, wouldn't want to undo it

Reply:
This is just one of the layers for building the react app; reduces the number of vulns. The final image is still bullseye. The bundle is copied over to the bullseye image. It doesn't seem to me what intermediate images matter.

Reply:
Ah, I see. That makes sense then!

Reply:
@epicfaace, can you approve?